docs/src/main/paradox/discovery.md

```diff
@@ -521,6 +569,53 @@ Notes:
 
 * If Akka management port tag is not found on service in Consul the implementation defaults to catalog service port.
 
+## Discovery Method: Aggregate multiple discovery mechanisms
+
+Aggregate discovery allows multiple discovery mechanisms to be aggregated e.g. try and resolve
+via DNS and fall back to configuration.
+
+To use aggregate discovery add its dependency as well as all of the discovery mechanisms that you 
+want to aggregate.
+
+@@dependency[sbt,Gradle,Maven] {
+  group="com.lightbend.akka.discovery"
+  artifact="akka-discovery-aggregate_2.12"
+  version="$version$"
+}
+
+Configure `akka-aggregate` as `akka.discovery.method` and which discovery mechanisms are tried and in which order.
+
+```
+akka {
+  discovery {
+    method = akka-aggregate
+    akka-aggregate {
+      discovery-mechanisms = ["akka-dns", "akka-config"]
+    }
+    akka-config {
+      services = {
```

Comment:
would prefer ` { ` rather than ` = {` for maps

Reply:
Yep